lxd/storage/backend_lxd.go

```diff
@@ -5157,6 +5162,11 @@ func (b *lxdBackend) UpdateCustomVolume(projectName string, volName string, newD
 			return fmt.Errorf("Custom volume 'block.filesystem' property cannot be changed")
 		}
 
+		// Check that the volume's volatile.uuid property isn't being changed.
+		if changedConfig["volatile.uuid"] != "" {
+			return fmt.Errorf("Custom volume 'volatile.uuid' property cannot be changed")
```

Comment:
Please use double quotes for quoting vars in errors and use backticks for fmt.Errorf() to avoid having to escape them. This should be consistent with the error in UpdateInstance.

Reply:
Changed, also for the error above.